apps/api/src/server/app.js

```diff
@@ -4,14 +4,12 @@ import morgan from 'morgan';
 import helmet from 'helmet';
 import { homeRoutes } from './app/home/home.routes.js';
 import { validationRoutes} from './app/validation/validation.routes.js';
-
-
-const app = express();
-
 import swaggerUi from 'swagger-ui-express';
```

Comment:
The order of imports should be on top all that come from node_modules, then our own files.